app/experimenter/static/js/index.js

```diff
@@ -17,7 +17,8 @@ const branchesDiv = document.getElementById("react-branches-form");
 
 if (branchesDiv) {
   const slug = branchesDiv.dataset.experimentSlug;
-  const experimentType = branchesDiv.dataset.experimentType;
+  //const experimentType = branchesDiv.dataset.experimentType;
+  const experimentType = "multi-pref";
```

Comment:
Ah yep don't do it this way, just pass `is_multi_pref1` as a separate prop, look at teh way I did it for branched.